app/views/main.py

```diff
@@ -22,21 +22,21 @@
 logger = logging.getLogger(__name__)
 
 
-root_blueprint = Blueprint('root', __name__)
+main_blueprint = Blueprint('main', __name__)
 
 
-@root_blueprint.after_request
+@main_blueprint.after_request
 def add_cache_control(response):
     response.cache_control.no_cache = True
     return response
 
 
-@root_blueprint.route('/', methods=['GET'])
+@main_blueprint.route('/', methods=['GET'])
 def root():
     raise NotFound
 
 
-@root_blueprint.route('/information/<message_identifier>', methods=['GET'])
+@main_blueprint.route('/information/<message_identifier>', methods=['GET'])
```

Comment:
I think we can delete this. Instead of redirecting the to a different URL we could raise a MultipleSurveyError and handle it in errors.py. That would leave the only route as the `/session` route. That might solve our blueprint naming issue as we could call it authentication/session_blueprint